Mathlib/Topology/Covering.lean

```diff
@@ -27,43 +28,105 @@ variable {E X : Type*} [TopologicalSpace E] [TopologicalSpace X] (f : E → X) (
 
 /-- A point `x : X` is evenly covered by `f : E → X` if `x` has an evenly covered neighborhood. -/
 def IsEvenlyCovered (x : X) (I : Type*) [TopologicalSpace I] :=
-  DiscreteTopology I ∧ ∃ t : Trivialization I f, x ∈ t.baseSet
```

Comment:
Could you add a comment somewhere about how it might be nice to later refactor `Trivialization` to work nice with empty types, in which case this can be switched back?

Reply:
I added remark, but I think it's unlikely we could refactor PartialHomeomorph while preserving most APIs ... If the APIs are not preserved then switching back would again be massive work.